src/plugins/vis_types/timeseries/public/application/components/aggs/invalid_agg.tsx

```diff
@@ -23,6 +24,10 @@ interface InvalidAggProps {
 
 export const getInvalidAggComponent =
   (message: JSX.Element | string) => (props: InvalidAggProps) => {
+    const { euiTheme } = useEuiTheme();
+    const titleStyles = css`
+      margin-top: -${euiTheme.size.xs};
+    `;
```

Comment:
This same style is used three different times in this PR. Can likely be consolidated in the `common.styles.ts` file.

Reply:
done